src/main/kotlin/money/Amount.kt

```diff
@@ -0,0 +1,21 @@
+package money
+
+import java.lang.IllegalArgumentException
+
+class Amount(private val stringAmount: String) {
+    private val integers = '0'..'9'
+
+    init {
+        if (stringAmount.isBlank()) {
+            throw IllegalArgumentException("값이 비어있습니다.")
+        }
+
+        if (!isInteger(stringAmount)) {
+            throw IllegalArgumentException("입력한 $stringAmount 값은 숫자여야 합니다.")
+        }
```

Comment:
```suggestion
        require(stringAmount.isNotBlank()) { "값이 비어있습니다." }
        require(isInteger(stringAmount)) { "입력한 $stringAmount 값은 숫자여야 합니다." }
```

Kotlin 표준 라이브러리는 이러한 목적으로 `check()`, `require()`, `checkNotNull()` 및 `requireNotNull()` 함수를 제공합니다. 특별히 커스텀된 에러를 던져야하는게 아니라면 이런 빌트인 기능으로 에러를 다루는게 전체적인 일관성을 위해 권장됩니다~

+ Java 와는 달리 `isNotBlank()` 가 기본 인터페이스로 포함되어 있는 것도 이런 검증 로직 가독성에 도움을 주는 것 같네요

https://kotlinlang.org/api/latest/jvm/stdlib/kotlin/check.html

Reply:
빌트인된 기능에 대해서 좀 익숙해지면 정말 편하겠네요, IDE 상에서도 추천 suggestion이 뜨는 거 같군요.

경근님 리뷰 덕에 많이 배웁니다!